install_config/install/prerequisites.adoc

```diff
@@ -79,7 +79,13 @@ back end; see xref:configuring-docker-storage[Configuring Docker Storage]
 below.
 
 |External etcd Nodes
-|Minimum 20 GB hard disk space for etcd data.
+a|* Minimum 20 GB hard disk space for etcd data.
+* Consult
+link:https://github.com/coreos/etcd/blob/master/Documentation/op-guide/hardware.md#hardware-recommendations[Hardware
+Recommendations] to properly size your etcd nodes.
```

Comment:
The information in this link is great, however I don't think it looks good for us to point customers at github wiki page to define such an important concept.